### ISSUE TITLE
[Vehicle] Seamoth docking sync

### ISSUE THREAD

**Issue by Sunrunner37** [SubnauticaNitrox/Nitrox#118]
- [ ] Docking inside cyclops
- [ ] Docking inside moon pool

**Comment by hamsteyr** [comment]
Docking seems to work, but undocking still seems to have an issue with syncing.

Replicable.

**Comment by Agent77326** [comment]
In my case neither did work, tested with 2 players and 1.0.0.9 from today.

**Comment by Aocxiez** [comment]
Case close thank you for your hardwork.

**Comment by kinvaris** [comment]
TLDR: we reproduced it twice on a different server with same versions. 

Same issue occured during 2 players with Nitrox version `Alpha 1.1.0.0`. We were playing on september build of 2018. 

During the first time, we thought we could not fix it. The bug starts when you try to dock the seamoth into the moonpool or cyclops. 

Every player is immediately desynced. The desync can only be fixed by destroying the moonpool/cyclops and performing a re-login into the server. (This we noticed during our second playthrough.)

The details of the Subnautica build to download via steam console: `download_depot 264710 264712 5196577974721678848 :`

App id: 264710
Depot id: 264712
Manifest id (version of build): 5196577974721678848

Reference Subnautica build in steamDB: https://steamdb.info/depot/264712/manifests/

**Comment by acidscorch** [comment]
Please retest the moonpool related part after upcoming fix for sync and store moonpool Ids.
